aws-test/tests/aws_ec2_autoscaling_group/test-hydrate-expected.json

```diff
@@ -4,7 +4,7 @@
     "policies": [
       {
         "AdjustmentType": "ChangeInCapacity",
-        "Alarms": null,
+        "Alarms": [],
```

Comment:
@ParthaI Is this change in expected.json because the data format changed, or because it was always `null` and our test is incorrect?

Reply:
@cbruno10 It is changed due to the data format changed in AWS SDK V2.